modules/mono/build_scripts/build_assemblies.py

```diff
@@ -216,6 +216,8 @@ def build_godot_api(msbuild_tool, module_dir, output_dir, push_nupkgs_local):
         args = ["/restore", "/t:Build", "/p:Configuration=" + build_config, "/p:NoWarn=1591"]
         if push_nupkgs_local:
             args += ["/p:ClearNuGetLocalCache=true", "/p:PushNuGetToLocalSource=" + push_nupkgs_local]
+        if float_size == 64:
+            args += ["/p:ExtraDefineConstants=REAL_T_IS_DOUBLE"]
```

Comment:
You forgot to update it here.

```suggestion
        if float_size == "64":
            args += ["/p:GodotFloat64=true"]
```

Reply:
Whoops, thanks